pyccel/codegen/codegen.py

```diff
@@ -207,52 +186,13 @@ def _collect_statements(self):
     def _set_kind(self):
```

Comment:
can we remove this, and move this line to init method,
then use is_program for checking if it is a program?

Reply:
Fixed 0eb7808c6b4c02ff245f9683cf18d58fa8b6ca01